apps/docs/README.md

```
@@ -0,0 +1,47 @@
+
+Run development server:
+
+```bash
+npm run dev
+# or
+pnpm dev
+# or
+yarn dev
+```
+
+Open http://localhost:3000 with your browser to see the result.
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)